xml/ha_clvm.xml

```diff
@@ -348,6 +345,139 @@ cLVM) for more information and details to integrate here - really helpful-->
   /dev/sdb vgtest b</screen>
     </step>
    </procedure>
+   <para>To test your clustered mirror device, proceed as follows:</para>
```

Comment:
In original section, we only use 2 devices for raid10 exmaple. Now, we just need to expand to 4 devices. For example,
"""
use the following procedure (assuming that <filename>/dev/sda</filename>
and <filename>/dev/sdb</filename> are the shared storage devices):
"""
we need 2 more devices here - /dev/sdc, /dev/sdd should be OK.